test/testnet/localnet/Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: deploy stop update-prereq build-tendermint status
+
+ACTIVATE_VENV := source venv/bin/activate
+TESTNET ?= tik
+
+build-tendermint:
+	@echo "Building Tendermint binary for Linux..."; \
+	cd ${GOPATH}/src/github.com/tendermint/tendermint; \
+	make build-linux; \
+	echo "Done."
+
+# Sets up a clean instance of this test network and fires it up
+deploy: venv venv/bin/ansible
+	@$(ACTIVATE_VENV); \
```

Comment:
Maintaining ansible installation will become a hassle very quickly. (Especially because it's hard to do it across different operating systems.) Let's put up a big warning that ansible is a pre-requisite requirement and let the user decide how they install it. For example most of our developers already have it using `brew install ansible` which they know how to maintain, while they might never have seen Python virtualenv before. As much as I like Python, I think we should abstain from it in this Makefile.